src/components/PatientProvider.tsx

```diff
@@ -3,16 +3,19 @@ import { useFHIRClient } from './FHIRClient';
 
 interface PatientProviderProps {
   children: ReactNode;
+  demoPatient?: fhir.Patient;
 }
 
 export const PatientContext = createContext<fhir.Patient | null>(null);
 
-export const PatientProvider: FC<PatientProviderProps> = ({ children }) => {
+export const PatientProvider: FC<PatientProviderProps> = ({ children, demoPatient }) => {
   const client = useFHIRClient();
-  const [patient, setPatient] = useState<fhir.Patient | null>(null);
+  const [patient, setPatient] = useState<fhir.Patient | null>(demoPatient || null);
 
   useEffect(() => {
-    client.patient.read().then((patient: fhir.Patient) => setPatient(patient));
+    if (client && client.patient.read) {
```

Comment:
Use optional chaining
```suggestion
    if (client?.patient?.read) {
```
You might be able to get away without the if statement too using this directly with `client?.patient?.read().then((patient: fhir.Patient) => setPatient(patient));`